homeassistant/components/blue_current/entity.py

```diff
@@ -0,0 +1,48 @@
+"""Entity representing a Blue Current charge point."""
+from homeassistant.const import ATTR_NAME
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+from . import Connector
+from .const import DOMAIN, MODEL_TYPE
+
+
+class BlueCurrentEntity(Entity):
+    """Define a base charge point entity."""
+
```

Comment:
```suggestion

    _attr_has_entity_name = True

```
This way you can omit this from all the entity descriptions in sensor.py